x-pack/package.json

```diff
@@ -103,12 +122,14 @@
     "run-sequence": "^2.2.1",
     "sass-loader": "^7.1.0",
     "simple-git": "1.37.0",
-    "sinon": "^5.0.7",
+    "sinon": "^7.2.2",
     "supertest": "^3.1.0",
     "supertest-as-promised": "^4.0.2",
     "tmp": "0.0.31",
     "tree-kill": "^1.1.0",
+    "ts-jest": "^23.1.4",
```

Comment:
x-pack doesn't actually integrate with ts-jest directly, it requires the ts-jest integration from `src/dev` in the kibana part of things, so this dependency shouldn't be necessary.

Reply:
done